benchmarks/ts/suites/trees.ts

```diff
@@ -0,0 +1,50 @@
+import b from "benny";
+import { IncrementalQuinTree, hash5 } from "maci-crypto";
+
+import { OptimisedMT } from "../src/optimisedmt";
+
+const name = "merkle-trees";
```

Comment:
It's better to use uppercase for such constants

Reply:
sure